xml/ay_custom_user_scripts.xml

```diff
@@ -430,7 +433,9 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
          <para>
           Optional; default is the type of the script (pre-scripts in this
           case). If you have more than one script, you should define
-          different names for each script.
+          different names for each script. If it is not defined and
```

Comment:
> If it is not defined and

If **what** is not defined?

Reply:
the filename option. I am also not sure how it will look like when para is so long.